frontend/src/components/shared/ComparisonDetails.vue

```diff
@@ -16,9 +16,8 @@
           <router-link :to="{ name: 'explorer', params: { model: currentModel.short_name } }">
             <b>{{ currentModel.short_name }}</b>
           </router-link>
-          has
-          {{ comparisonDetails.details['Reaction'].own }} reactions and
-          {{ comparisonDetails.details['CompartmentalizedMetabolite'].own }} metabolites.
+          v{{ currentModel.version }} has {{ comparisonDetails.details['Reaction'].own }} reactions
```

Comment:
It looks like Human-GEM is not including the `v`:
<img width="173" alt="image" src="https://user-images.githubusercontent.com/23480589/165438160-b06c37cc-0361-454f-bfc1-fe8be8ff4255.png">

```suggestion
          {{ currentModel.version }} has {{ comparisonDetails.details['Reaction'].own }} reactions
```

Reply:
There is still a `v` here though. 
![modelvname](https://user-images.githubusercontent.com/1029190/165447960-83b84ab8-0394-41f8-bb7a-bdd98938dc8c.png)
Might be a bit trickier to get rid of:
![modelinfo](https://user-images.githubusercontent.com/1029190/165448057-97d4f6ef-d89b-4a53-a9bc-64114fb3b38a.png)

Reply:
NIce catch! It looks like we should have an issue to prune the unwanted `v`s. If there's anything that can be done in this PR without going to much out of scope, that's okay, but I would suggest leaving the bulk of the work for the separate issue.

Reply:
Turns out it was easy. Fixed in 598cec2. I also created #966.